packages/app/src/systems/Ecosystem/components/ProjectItem/ProjectItemLoader.tsx

```diff
@@ -0,0 +1,29 @@
+import { cssObj } from '@fuel-ui/css';
+import { Card, ContentLoader, type ContentLoaderProps } from '@fuel-ui/react';
+
+export const ProjectItemLoader = (props: ContentLoaderProps) => {
+  return (
+    <Card>
+      <ContentLoader
+        style={styles.loader}
+        height={160}
+        viewBox="0 0 496 180"
+        {...props}
+      >
+        <rect x="20" y="30" width="40" height="40" rx="4" />
+        <rect x="80" y="30" width="120" height="20" rx="4" />
+        <rect x="80" y="80" width="400" height="14" rx="4" />
+        <rect x="460" y="30" width="20" height="20" rx="4" />
+        <rect x="80" y="120" width="80" height="20" rx="4" />
+        <rect x="400" y="120" width="80" height="20" rx="4" />
```

Comment:
can we make the elements match better this format with 2 lines in the project description

<img width="472" alt="image" src="https://github.com/FuelLabs/fuels-portal/assets/8636507/5e17eb05-c5b5-4ad9-a9e3-08ca7da71513">